.github/workflows/build-rfc-pdf.yml

```
@@ -0,0 +1,62 @@
+          sudo apt-get update
+          sudo apt-get install -y pandoc \
+            texlive-xetex texlive-latex-extra texlive-fonts-recommended \
+            texlive-fonts-extra texlive-luatex texlive-plain-generic \
+            fonts-dejavu fonts-liberation
+          fc-cache -fv
+
+      - name: Install Mermaid CLI
+        run: npm install -g @mermaid-js/mermaid-cli
+
+      - name: Optional clean
+        if: ${{ github.event.inputs.clean == 'true' }}
```

actionlint (1.7.7)

property "clean" is not defined in object type {}

(expression)